app/domain_models/record_sets_create.rb

```diff
@@ -0,0 +1,38 @@
+class RecordSetsCreate
```

Comment:
Domain model should include the domain knowledge.
Like, what contents should be registered as sets, and what contents should be registered as time, etc.